casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -236,7 +236,11 @@ object Genesis {
         }
       }
       .flatMap {
-        case Success(bonds) => bonds.pure[F]
+        case Success(bonds) =>
+          bonds.toList.traverse_ {
+            case (pk, stake) =>
+              Log[F].info(s"Created validator $pk with bond $stake")
```

Comment:
Nitpick: `PrettyPrinter.buildString(pk)` and perhaps replace "created" with "Parsed"